perception/radar_tracks_msgs_converter/config/config-radar_tracks_msgs_converter.param.yaml

```diff
@@ -0,0 +1,5 @@
+/**:
+  ros__parameters:
+    update_rate_hz: 20.0
+    new_frame_id: "base_link"
+    use_twist_compensation: false
```

Comment:
Would you update the Readme as well so that it will automatically reads the schema file?

Reply:
Done.